src/plugins/intel_npu/tools/single-image-test/main.cpp

```diff
@@ -97,6 +97,7 @@ DEFINE_string(data_shape, "",
     "In case of one input size: \"[1,3,224,224]\"");
 DEFINE_string(skip_output_layers, "" , "Skip output layers from the network. Currently only applicable for"
         "RRMSE and NRMSE mode. Accept ';' separated list of output layers");
+DEFINE_bool(clamp_u8, false, "Apply clamping when convert fp to u8 to make models like edsr work correctly.");
```

Comment:
We shouldn't mention specific networks.
```suggestion
DEFINE_bool(clamp_u8, false, "Apply clamping when converting FP to U8");
```

Reply:
Fixed in [384f801](https://github.com/openvinotoolkit/openvino/pull/29592/commits/384f801021c8bed64467cc8cee19d48354af27ac)